docs/release-process.md

```diff
@@ -0,0 +1,184 @@
+
+In this repo, the build automation is triggered for each commit to the `main` branch.
+
+This task runs using a GitHub access token tied to Eric's `@scouten-adobe` account, which is why all PRs and releases are under Eric's name. (See ticket CAI-7621 for moving this to a team-owned bot account.)
+
+### Create a release pull request
+
+This runs [`release-plz release-pr`](https://release-plz.dev/docs/usage/release-pr). For each published crate in the repo, it looks for commits since the last tagged release (ignoring commits with the `chore:` prefix). If it finds any, it will open a new release pull request or update the existing release. (NOTE: When Colin's [PR #2196 to release-plz](https://github.com/release-plz/release-plz/pull/2196) lands there will be a new trigger here which will reference intra-project dependency updates even if the downstream project has no commits of its own.)
+
+For each project that is to be updated, release-plz will generate the following changes:
+
+* `cargo.toml`: **Update version number.**
+  * If any API breaking changes are detected, bump the minor version number. (This will change to a major version number once we do 1.0 release.)
+  * If any feature changes (API additions) are detected, bump the minor version number. (This will _not_ change when we do 1.0 release.)
+  * If only bug-fix changes are detected, bump the patch version number.
+  * For all crates, `release-plz` will evaluate the commit history since the previous release tag using [Conventional Commit syntax](https://www.conventionalcommits.org/en/v1.0.0/#summary) to determine API breaking changes, feature changes, or only bug-fix changes.
+  * For library crates _only,_ `release-plz` will also download the most recent release from [crates.io](https://crates.io) to verify any API surface changes. The larger scope (patch -> minor -> major) will be used to select the new version number.
+
+* `cargo.toml`: **Update version references for other crates in the same repo.**
+  * For each dependency that also exists in this repo, update the version number to match what is generated for that project.
+
+* `CHANGELOG.md`: **Add description of new release.**
+  * This adds a new section to the top of the changelog file with a link to the diffs between this version and the previous release.
+  * The changelog section also contains a list of commits that affected this crate's source, grouped by the commit type. (See [Conventional Commit syntax](https://www.conventionalcommits.org/en/v1.0.0/#summary).)
+
+IMPORTANT: It _is_ permissible to manually edit the proposed changelog in the PR, but be aware those changes will be overwritten if another update is triggered. release-plz uses force-push to update any existing release PR.
+
+IMPORTANT: If the version number of the crate in `main` is _different_ from the version number on [crates.io](https://crates.io), this job will NOT run. (See the next task.)
```

Comment:
`(See the next task.)` being the `Publish new crates` one below?

Reply:
Replace the "next task" reference with an explicit title.